api/server/server.go

```diff
@@ -20,12 +20,20 @@ import (
 
 type ToDDApi struct {
 	cfg config.Config
+	tdb db.DatabasePackage
 }
 
-func (tapi ToDDApi) Start(cfg config.Config) {
+func (tapi ToDDApi) Start(cfg config.Config) error {
 
 	tapi.cfg = cfg
 
+	tdb, err := db.NewToddDB(tapi.cfg)
+	if err != nil {
+		return err
```

Comment:
Would like to see something a little different here. See: https://twitter.com/davecheney/status/697955801806209024 (Yes I just used Twitter in a citation :smile: )